CHANGELOG.md

```diff
@@ -1,6 +1,13 @@
 Changelog
 =========
 
+## [1.18.0] - 2020-03-30
+### Added
+ * relay: Include time since last activity in dropped call log (#774)
+ * connection: Separate out lastActivity into recv and send activity (#770)
```

Comment:
maybe call out that introspection tracks last activity separately for read/write (want to think about user impact in the changelog)

Reply:
Good call. Thanks!